ubtue_Collectanea Christiana Orientalia.js

```diff
@@ -0,0 +1,177 @@
+{
+	"translatorID": "f24cd419-5892-450a-b39e-be2ff3191757",
+	"label": "Collectanea Christiana Orientalia",
+	"creator": "Timotheus Kim",
+	"target": "/issue/view/",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2020-05-14 12:57:39"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2019 Universitätsbibliothek Tübingen.  All rights reserved.
+
+	This program is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	This program is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+function detectWeb(doc, url) {
+	if (url.match(/\/issue\/view/))
+		return "multiple";
+	else if (url.match(/article/)) {
+		// placeholder, actual type determined by the embedded metadata translator
+		return "journalArticle";
+	}
+}
+
+function getSearchResults(doc) {
+	var items = {};
```

Comment:
please use let instead of var for items, found, rows

Reply:
https://github.com/socheres/translators/blob/master/ubtue_Collectanea%20Christiana%20Orientalia.js